django_project/certification/models/course_convener.py

```diff
@@ -36,6 +36,20 @@ class CourseConvener(models.Model):
         blank=True
     )
 
+    title = models.CharField(
```

Comment:
Your code should not be touching this file, not sure why it is part of this changes. Please roll it back to the original, and make sure you have rebased develop into your working branch.